src/main/resources/templates/pps/home.html

```diff
@@ -14,105 +14,97 @@
-                    <li>to sign in to or create a Companies House account, which is different from a WebFiling account</li>
+                    <li>to sign in to or create a Companies House account (this is different to a WebFiling account)
+                    </li>
                     <li>a credit or debit card</li>
+                    <li>the information on your penalty notice (that we sent you in the post)</li>
                 </ul>
 
+                <div class="govuk-!-margin-top-6">
+                    <details class="govuk-details">
+                        <summary class="govuk-details__summary">
+                            <span class="govuk-details__summary-text">I've lost my penalty notice
+                            </span>
+                        </summary>
+                        <div class="govuk-details__text">
+                            <p class="govuk-body"> Email <a href="mailto:enquiries@companieshouse.gov.uk"
+                                                            class="govuk-link">enquiries@companieshouse.gov.uk</a> to
+                                let us know if you have lost your penalty notice. Use 'Lost penalty notice' as the email
+                                subject.
+                            </p>
+                            <p></p>Tell us as much information as you can about the penalty so we can pass your message
+                            to the right team.
+                        </div>
+                    </details>
+                </div>
+
+                <p class="govuk-body">You must pay the full amount.</p>
+                <p class="govuk-body">It takes around 5 minutes to pay.</p>
+
+                <a href="what-does-the-penalty-reference-start-with" role="button" draggable="false"
+                   class="govuk-button govuk-button--start" data-module="govuk-button" data-event-id="pps - Start now"
+                   onclick="_paq.push(['trackGoal', 3]);">Start now
+                    <!--Waiting for SAN-29 to complete link above (line 50)-->
+                    <svg class="govuk-button__start-icon" xmlns="http://www.w3.org/2000/svg" width="17.5" height="19"
+                         viewBox="0 0 33 40" aria-hidden="true" focusable="false">
+                        <path fill="currentColor" d="M0 0h13l20 20-20 20H0l20-20z"></path>
+                    </svg>
+                </a>
+
                 <p class="govuk-body">
-                    You can
-                    <a class="govuk-link govuk-link--no-visited-state" id="appeals-link"
-                       data-event-id="lfp - appeal a penalty link" href="/appeal-a-penalty/">
-                        appeal a late filing penalty</a>
-                    , but you'll need to show that the reasons for filing late were exceptional.
+                    <a href="https://www.gov.uk/government/publications/companies-house-approach-to-financial-penalties"
```

Comment:
Might be job for separate ticket so can leave this full url in for now, but wondering if it's worth using env prop for the govUrl in the places where we use it -- similar to the churl we have.